demo/src/main.tsx

```diff
@@ -25,21 +25,21 @@ const clientId = (function () {
 
 // use this for local development with local realtime
 //
-// const ablyClient = new Ably.Realtime({
-//   authUrl: `/api/ably-token-request?clientId=${clientId}`,
-//   port: 8081,
-//   environment: 'local',
-//   tls: false,
-//   clientId,
-// });
-
 const realtimeClient = new Ably.Realtime({
   authUrl: `/api/ably-token-request?clientId=${clientId}`,
-  restHost: import.meta?.env?.VITE_ABLY_HOST ? import.meta.env.VITE_ABLY_HOST : undefined,
-  realtimeHost: import.meta?.env?.VITE_ABLY_HOST ? import.meta.env.VITE_ABLY_HOST : undefined,
+  port: 8081,
+  environment: 'local',
+  tls: false,
   clientId,
 });
 
+// const realtimeClient = new Ably.Realtime({
```

Comment:
I'm guessing this was just missed, but should probably be reverted? :)

Reply:
yup